styles/globals.css

```diff
@@ -14,3 +14,20 @@ a {
 * {
   box-sizing: border-box;
 }
+
+h3 {
+  font-size: 2em;
+  margin-bottom: 20px;
+}
+
+.MuiTypography-body1 {
+  margin-top: 16px;
```

Comment:
you can just do margin: 16px 0; to apply the same margin to the top and bottom!:)